packages/components/src/DateTimePickers/shared/common.scss

```diff
@@ -0,0 +1,20 @@
+$border-default: 0.1rem solid $gallery;
+
+@mixin main {
+    color: $dove-gray;
+    font-weight: 600;
+}
```

Comment:
why not using global className ?

Reply:
Because need more class inclusion in code ans is therefore more difficult to track what style is applied.
&#10005;